ui/src/shared/components/cells/Cells.tsx

```diff
@@ -119,5 +141,9 @@ class Cells extends Component<Props & WithRouterProps> {
     }
   }
 }
-
-export default withRouter(Cells)
+const mstp = (state: AppState): StateProps => {
+  return {state}
+}
+export default withRouter<OwnProps>(
```

Comment:
Since we don’t seem to be using redux, we can remove the connect here

Reply:
got it